wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/bootstrap/AbstractBootstrapper.java

```diff
@@ -86,6 +86,9 @@ public void bootstrap() throws InfrastructureException {
         if (finishEventFuture != null) {
             throw new IllegalStateException("Bootstrap method must be called only once.");
         }
+        if (!hasInstallersToBootstrap()) {
```

Comment:
i would do both
1. change the contract of the bootstrapper, but rather to throw exception when there is nothing to install, instead of silent quit. What if we do it directly in AbstractBootstrapper constructor? So the contract is that bootstrapper cannot be created with empty list of installers.
2. change the code in internal runtimes(like Sergii suggests) so bootstrapper is not created at all.